.changeset/thick-carrots-care.md

```diff
@@ -0,0 +1,38 @@
+---
+'@openfn/language-kobotoolbox': patch
+---
+
+Update `http.request` option docs to use `data` instead of `body` in requests
+
+
+### Migration Guide
```

Comment:
This migration guide will need moving as well then?

Reply:
@josephjclark  done